scripts/ai2-internal/run_with_beaker.py

```diff
@@ -66,56 +61,77 @@ def main(param_file: str, extra_beaker_commands: List[str]):
             "--file-friendly-logging"
         ]
 
-    # TODO(michaels): add back in the env list.
-    # Presently this makes the Beaker UI unusably cluttered.
-    command = [
-            '/usr/local/bin/beaker',
-            'experiment',
-            'run',
-            '--result-path',
-            '/output',
-            "--source",
-            f"{config_dataset_id}:/config.json"] + env + extra_beaker_commands + [image] + allennlp_command
-    print(' '.join(command))
-    subprocess.run(command, check=True)
+    dataset_mounts = []
+    for source in (args.source if args.source else []):
+        datasetId, containerPath = source.split(":") + "\n"
+        dataset_mounts.append({
+            "datasetId": datasetId,
+            "containerPath": containerPath
+        })
+    for source in [f"{config_dataset_id}:/config.json"]:
+        datasetId, containerPath = source.split(":")
+        dataset_mounts.append({
+            "datasetId": datasetId,
+            "containerPath": containerPath
+        })
+
+    for var in (args.env if args.env else []):
+        key, value = var.split("=")
+        env[key] = value
+
+    config_spec = {
+        "description": args.desc,
+        "blueprint": blueprint,
+        "resultPath": "/output",
+        "args": allennlp_command,
+        "datasetMounts": dataset_mounts,
+        "requirements": {
+            "cpu": float(args.cpu),
+            "memory": args.memory,
+            "gpuCount": args.gpu_count if args.gpu_count else 1,
+        },
+        "env": env
+    }
+    config_task = {"spec": config_spec}
```

Comment:
You may consider adding `"name": "<some value>"` here.  While optional, it would slightly simplify how this displays in the UI by hiding the task ID.

Reply:
Done.